From a3be778d1c4137d30af6ed9a08d0b54ee828687f Mon Sep 17 00:00:00 2001
From: madomado <madonuko@outlook.com>
Date: Tue, 21 Nov 2023 19:45:14 +0800
Subject: [PATCH] fix(wallpaper): change wallpaper for dark theme settings

Previously before the fix, the wallpaper cannot be changed via settings because it only sets the wallpaper for the light theme, not the dark theme.
This fixes the issue.
---
 src/Views/Wallpaper.vala | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/Views/Wallpaper.vala b/src/Views/Wallpaper.vala
index 07251444..7141cad8 100644
--- a/src/Views/Wallpaper.vala
+++ b/src/Views/Wallpaper.vala
@@ -206,6 +206,7 @@ public class PantheonShell.Wallpaper : Gtk.Box {
         }
 
         gnome_background_settings.set_string ("picture-uri", uri);
+        gnome_background_settings.set_string ("picture-uri-dark", uri);
     }
 
     private void update_checked_wallpaper (Gtk.FlowBox box, Gtk.FlowBoxChild child) {
